src/forwarder/internal/pkg/authz/write.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 Cisco and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package authz has convenience functions for getting authz policy
+package authz
+
+import (
+	"context"
+	"io/ioutil"
+
+	"github.com/open-policy-agent/opa/rego"
+
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+)
+
+const (
+	// AuthzRegoFilename - default file containing authz rego policy
+	AuthzRegoFilename = "/etc/nsm/authz.rego"
```

Comment:
Is this policy missed? Do not see authz.rego in these changes.

Reply:
I did't see this file in 'cmd-forwarder-vppagent', so I thought that the user creates this file.

Reply:
ok, then let's remove this const